### PR TITLE
qt@g.o: fdo-mime->xdg-utils

### PR DESCRIPTION
**Pull request by hanetzer**

### CODE REVIEW

File: mail-client/trojita/trojita-0.7-r2.ebuild [hunk 1]
```diff
@@ -27,8 +27,8 @@ RDEPEND="
 	dev-qt/qtwebkit:5
 	dev-qt/qtwidgets:5
 	crypt? (
-		>=app-crypt/gpgme-1.8.0[cxx,qt5]
 		dev-libs/mimetic
+		>=app-crypt/gpgme-1.8.0[cxx,qt5]
```
[the remark this answers is not in the document]
Comment (hanetzer) [reply mail-client/trojita/trojita-0.7-r2.ebuild]: @a17r prolly fat-fingerd on vimdiff, gimme a sec and I'll fix that.

[the remark this answers is not in the document]
Comment (hanetzer) [reply mail-client/trojita/trojita-0.7-r2.ebuild]: Ah no, this is not that. Basically its keeping the live and versioned ebuilds as similar as possible.

[the remark this answers is not in the document]
Comment (a17r) [reply mail-client/trojita/trojita-0.7-r2.ebuild]: if that's the case then fix ordering in the live ebuild instead.

**Comment by jktjkt** [comment]
As a proxy maintainer with no knowledge of the fdo/xdg eclasses, the changes to trojita look reasonable. If I was doing them, I would split the actual changes of fdo/xdg into once commit and the unification into another commit.

File: mail-client/trojita/trojita-0.7-r2.ebuild (hunk 1 above)
[the remark this answers is not in the document]
Comment (hanetzer) [reply mail-client/trojita/trojita-0.7-r2.ebuild]: Done, and I went ahead and moved the `""->()` change to another commit along with the change.

**Comment by hanetzer** [comment]
@jktjkt Gotcha, did so.

**Pull request closed without merging**